src/main/java/Duke.java

```diff
@@ -1,10 +1,65 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final int MAX_SIZE = 100;
+    private static Task[] userTasks = new Task[MAX_SIZE];
+    private static int userTasksCount = 0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String userInput;
+        Scanner in = new Scanner(System.in);
+
+        System.out.println("Hello! I'm Duke" + System.lineSeparator() + "What can I do for you?");
+
+        userInput = in.nextLine();
```

Comment:
Hi Jericho. Your code is readable and there are no other serious issues. However, I would like to suggest that you do not need to have the spaces in lines 11 and 13. I think that your code would still be readable even without these spaces. However, if you do feel that these spaces improves readability, do implement them.